js-watcher/cache.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"path/filepath"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/dynamic/dynamicinformer"
+	"k8s.io/client-go/rest"
+	k8sCache "k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+type Cache struct {
+	lister    k8sCache.GenericLister
+	informer  k8sCache.SharedIndexInformer
+	namespace string
+	userIdLabel string
```

Comment:
not `gofmt` compliant - at least run it through that. I'd also recommend addressing `revive` comments generally, perhaps not the comment ones but at least the ones on variable and function names....

Reply:
ran it through gofmt and revive and cleared all errors